src/services/files.service.js

```diff
@@ -0,0 +1,66 @@
+import multer from 'multer';
+import fs from 'fs';
+import aws from 'aws-sdk';
+import config from '../config';
+
+/**
+ * Files service
+ */
+class fileService {
+  /**
+  * @param {String} field
+  * @returns {object} file
+  */
+  upload(field) {
+    const dir = 'public/images';
+
+    if (!fs.existsSync(dir)) {
+      fs.mkdirSync(dir);
+    }
+    const storage = multer.diskStorage({
+      destination: (req, file, cb) => {
+        cb(null, 'public/images');
+      },
+      filename: (req, file, cb) => {
+        cb(null, `${Date.now()}-${file.originalname}`);
+      }
+    });
+    return multer({
```

Comment:
I like how you handled file upload, Great job @nshaphan , I have tested and I think this PR is ready to be merged